docs/trusted-activities.yml

```diff
@@ -0,0 +1,193 @@
+          schema:
+            $ref: '#/definitions/TrustResourceRequest'
+      responses:
+        200:
+          description: On Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/TrustedResourceResponse'
+      deprecated: false
+  /api/v1/trusted-activities/{id}:
+    get:
+      tags:
+        - trust-resource-controller
+      summary: Gets a single trusted activity based on the resource value provided
+      operationId: getTrustResourceUsingGET
+      parameters:
+        - name: id
+          in: path
+          description: id
+          required: true
+          schema:
+            type: integer
+            format: int32
+      responses:
+        200:
+          description: On Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/TrustedResourceResponse'
+      deprecated: false
+    put:
+      tags:
+        - trust-resource-controller
+      summary: Modify a trusted activity
+      operationId: editTrustResourceUsingPUT
+      consumes:
+        - application/json
+      parameters:
+        - name: id
+          in: path
+          description: id
+          required: true
+          type: integer
+          format: int32
+        - in: body
+          name: request
+          description: request
+          required: true
+          schema:
+            $ref: '#/definitions/TrustResourceRequest'
+      responses:
+        200:
+          description: On Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/TrustedResourceResponse'
+      deprecated: false
+    delete:
+      tags:
+        - trust-resource-controller
+      summary: Delete a trusted activity
+      operationId: deleteTrustResourceUsingDELETE
+      parameters:
+        - name: id
+          in: path
+          description: id
+          required: true
+          type: integer
+          format: int32
+      responses:
+        204:
+          description: On Success
+          content:
+            application/json:
+              schema:
+                type: object
+      deprecated: false
+definitions:
+  TrustResourceRequest:
+    type: object
+    required:
+      - type
+      - value
+    properties:
+      type:
+        type: string
+        example: DOMAIN
+      value:
+        type: string
+        example: test-domain.com
+      description:
+        type: string
+        nullable: true
+    title: TrustResourceRequest
+  TrustedResourceResponse:
+    type: object
+    properties:
+      description:
+        type: string
+        description: The description of the trusted resource.
+        nullable: true
+      resourceId:
+        type: string
+        description: The identifier of the resource.
+        example: "123"
+      type:
+        type: string
+        description: 'The type of the trusted resource. eg: SLACK, DOMAIN, IP_ADDRESS.'
```

Comment:
Isn't `IP_ADDRESS` part of a different endpoint?

Reply:
This is generated from their API swagger docs... currently ip-addresses are implemented at all in the console and it looks like the plan is to eventually add them as an additional type to these endpoints.  This doesn't exist yet so the `IP_ADDRESS` type isn't part of py42 yet.